spec/stack_master/parameter_loader_spec.rb

```diff
@@ -1,81 +1,105 @@
 RSpec.describe StackMaster::ParameterLoader do
-  subject(:parameters) { StackMaster::ParameterLoader.load(parameter_files) }
-  let(:parameter_files) { [
-    '/base_dir/parameters/stack_name.yml',
-    '/base_dir/parameters/us-east-1/stack_name.yml'
-  ] }
 
-  context "no parameter file" do
+  let(:stack_file_name) { '/base_dir/parameters/stack_name.yml' }
+  let(:region_file_name) { '/base_dir/parameters/us-east-1/stack_name.yml' }
+
+  subject(:parameters) { StackMaster::ParameterLoader.load([stack_file_name, region_file_name]) }
+
+  context 'no parameter file' do
     before do
-      allow(File).to receive(:exists?).and_return(false)
+      file_mock(stack_file_name, false)
+      file_mock(region_file_name, false)
     end
```

Comment:
Can this before block just be moved to the top level and then we can dry it up even more?